Javascript_2/Flight booking fullname function/script.js

```
@@ -0,0 +1,33 @@
+function getFullName(
+  firstName,
+  surName,
+  useFormalName = false,
+  isFemale = false
+) {
+  firstName = firstName.trim();
+  surName = surName.trim();
+
+  if (firstName === "" && surName === "") {
+    return "";
```

You can return an error message instead of an empty string.